src/ktn_dodger.erl

```diff
@@ -29,9 +30,9 @@
 %% expanding preprocessor directives and macro applications, as long as
 %% these are syntactically "well-behaved". Because the normal parse
 %% trees of the `erl_parse' module cannot represent these things
-%% (normally, they are expanded by the Erlang preprocessor (`epp')
```

Comment:
The `link' syntax should generated proper documentation with a link. Is it because we're not using ExDoc? (yet?)

Reply:
I have no idea… I just copied what `epp_dodger` had on OTP25 (I think). Then, I refrained to copy the newer docs (using `-moduledoc …` and such) because we still test the lib with OTP25, but eventually I will.